utils/countryUtils.js

```diff
@@ -1,39 +1,32 @@
 const stringUtils = require('./stringUtils');
 const countryData = require('./countries');
 
-/**
- * Transform null values to 0 (DEEPLY TRANSFORM)
- * @param {Object} object country/continent object
- * @returns  {Object} 		transformed object
- */
-const transformNull = (object) => {
-	if (typeof object !== 'object') {
-		return object;
+
+const convertNullToZero = (continentOrCountry = null) => {
+
+	const COUNTRY_INFO = 'countryInfo';
```

Comment:
this makes absolutely no sense, why should we have a constant variable for a value we use just once

Reply:
How could you test the value `countryInfo` in your unit test if you don't have a variable?

Reply:
just have a look at the tests, I don't see the connection between unittests and having a constant variable for a value that is used exactly once in that file

Reply:
Utility functions have not `unit tests` yet!